presidio_evaluator/evaluation/model_error.py

```diff
@@ -1,4 +1,5 @@
-from typing import Dict, List
```

Comment:
Changes here: improved the model error functionality to be able to extract more information for error analysis